### ISSUE TITLE
Host crash when changing to dark skin

### ISSUE THREAD

**Issue by saidbelkacem** [surge-synthesizer/surge#7035]
**Bug Description:**
Surge XT crashes Host (Reaper) when using dark skin. VST3 and LV2, same problem. Default skin works fine. This problem occurs only with the Plugins. The App version handles skins without problem.
So in Reaper I load a new instance of Surge on a new track. I can open and close without a problem. But as soon as I try to change to the dark skin and I close and try to reopen, the Host crashes.

**Surge XT Version**
Version: Surge XT 1.2.makepkg.bb8dd2b39
Build: 2023-05-05 @ 09:35:41 on 'archlinux/local' with 'GNU-13.1.1' using JUCE 6.1.6
System: Linux 64-bit LV2 on  AMD Ryzen 5 PRO 4650GE with Radeon Graphics
Sample Rate: 48.0 kHz / Process block size: 32 samples

Linux 6.3.4-arch1-1 #1 SMP PREEMPT_DYNAMIC Wed, 24 May 2023 17:44:00 +0000 x86_64 GNU/Linux

Reaper 6.79

**Comment by mkruselj** [comment]
This is probably caused by Arch changing compiler flags when it wasn't supposed to...

**Comment by baconpaul** [comment]
Well yeah maybe but it could be a variety of things including filesystem permissions. 

do you get a stack trace? I don’t run arch, and don’t know the build that leads to that version number. I’ll confirm 1.2 release works reaper ubuntu dark skin after the weekend but if you have a stack that would help

**Comment by baconpaul** [comment]
A few other questions also

from the about screen it shows a set of directories. Are those consistent between the daw and standalone version?

in the daw version can you load presets ok?

**Comment by mkruselj** [comment]
Yeah that build number suggests the build was made by some package maintainer and who the hell knows what they modified.

**Comment by baconpaul** [comment]
The hash is the 1.2.3 hash but 1.2.3 does t build with gcc 13 I don’t think so clearly something was changed. Some more details would really help us know what’s wrong

I tested in my Mac and could open and close the dark skin and like I said can check ubuntu next week unless someone on discord does a test for us this weekend

**Comment by saidbelkacem** [comment]
Uff, first, thank you for your fast responds. Second, thank you for fantastic work !! Absolutely love Surge.

And halleluja, solved. I just added my username to the group realtime.
The arch package "realtime-privileges" was already installed.
It creates this:
etc/security/limits.d/99-realtime-privileges.conf

Works like a charm now. I loaded the sober skin and I click around as I want, no more crashes
